developer-workflow/sbom.rst

```diff
@@ -107,3 +107,33 @@ When removing a dependency:
    that correct package is removed from the SBOM.
 5. Commit the changes to :cpy-file:`Misc/sbom.spdx.json` and
    :cpy-file:`Tools/build/generate_sbom.py`.
+
+Updating external dependencies (``cpython-source-deps``)
```

Comment:
It's a little bit awkward because the _intention_ is that it's not, but in practice, no other platform has needed it yet. (Maybe that's the clarification that is needed?)